Jenkinsfile

```diff
@@ -77,19 +57,21 @@ pipeline {
         not { expression { isPr } }
       }
       steps {
-        script { 
-          env.REGISTRY_PUSH = (params.RELEASE_BUILD) ? 'all' : 'internal'
-          env.SERVICE_VERSION = sbtbuild.getServiceVersion()
-          // set the SERVICE_SHA to the current head because it might not be the same as env.GIT_COMMIT
-          env.SERVICE_SHA = sh(returnStdout: true, script: "git rev-parse HEAD").trim()
-          env.DOCKER_TAG = dockerize.docker_build(env.SERVICE_VERSION, env.SERVICE_SHA, sbtbuild.getDockerPath(), sbtbuild.getDockerArtifact(), env.REGISTRY_PUSH)
+        script {
+          if (params.RELEASE_BUILD) {
+            env.REGISTRY_PUSH = (params.RELEASE_DRY_RUN) ? 'none' : 'all'
```

Comment:
Before, for release dry run, we were pushing the image to all registries.  Now we do not push it to any registries.